### ISSUE TITLE
Compile error on Ubuntu 16.04.2 x86_64

### ISSUE THREAD

**Issue by anlijun** [BradLarson/GPUImage2#230]
Compile error

$ swift --version
Swift version 4.0.3 (swift-4.0.3-RELEASE)
Target: x86_64-unknown-linux-gnu

$ ./compile-LinuxGL.sh
<unknown>:0: error: module 'SwiftGlibc' in AST file '/tmp/org.llvm.clang.ffff/ModuleCache/18MN48HZSQN76/SwiftGlibc-2AFGMHWHXQ37Q.pcm' (imported by AST file '/tmp/v4lfuncs-1a17a0.pch') is not defined in any loaded module map file; maybe you need to load '/home/ffff/toolchain/usr/lib/swift/linux/x86_64/glibc.modulemap'?
<unknown>:0: note: imported by '/tmp/v4lfuncs-1a17a0.pch'
<unknown>:0: error: clang importer creation failed
<unknown>:0: error: module 'SwiftGlibc' in AST file '/tmp/org.llvm.clang.ffff/ModuleCache/18MN48HZSQN76/SwiftGlibc-2AFGMHWHXQ37Q.pcm' (imported by AST file '/tmp/v4lfuncs-f2bbea.pch') is not defined in any loaded module map file; maybe you need to load '/home/ffff/toolchain/usr/lib/swift/linux/x86_64/glibc.modulemap'?
<unknown>:0: note: imported by '/tmp/v4lfuncs-f2bbea.pch'
<unknown>:0: error: clang importer creation failed

environment:
os: Ubuntu 16.04.2 LTS (GNU/Linux 4.4.0-98-generic x86_64)
swift: Swift version 4.0.3 (swift-4.0.3-RELEASE)

**Comment by BradLarson** [comment]
Yes, that build script is quite out of date. I have a new version that I've been working on using the Swift Package Manager, but haven't committed that yet.

**Comment by anlijun** [comment]
Waiting for you...
